packages/desktop-client/src/components/reports/graphs/BarGraph.tsx

```diff
@@ -241,30 +241,32 @@ export function BarGraph({
                     isAnimationActive={false}
                   />
                 )}
+                {!compact && <CartesianGrid strokeDasharray="3 3" />}
```

Comment:
Recharts + react v19 does not support using React.Fragment inside the graph definition. So I had to split the conditional into multiple smaller ones.